### PR TITLE
NITs raised in 1375

### PR DESCRIPTION
**Pull request by Adirio**
As requested in https://github.com/kubernetes-sigs/kubebuilder/pull/1375#pullrequestreview-362318348

NITs:
- https://github.com/kubernetes-sigs/kubebuilder/pull/1375#discussion_r382265396
- https://github.com/kubernetes-sigs/kubebuilder/pull/1375#discussion_r382274049

### CODE REVIEW

**Comment by Adirio** [comment]
@DirectXMan12 as requested in #1375, the NITs have been tackled here as the other PR was merged.

File: pkg/model/file/interfaces.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Template interface {
 	Builder
 	// GetBody returns the template body
 	GetBody() string
-	// SetTemplateDefaults returns the TemplateMixin for creating a scaffold file
+	// SetTemplateDefaults sets the default values for templates
```
Comment (camilamacedo86) [inline pkg/model/file/interfaces.go]: Nit which we are tryng to address here: 

Screenshot 2020-02-25 at 12 57 11

However, I could not find any implementation of `SetTemplateDefaults` which is returning an error actually. So, hould not bee better we change it  as follows:

```suggestion
	// SetTemplateDefaults sets the default values for templates
	SetTemplateDefaults() 
```

File: pkg/model/file/marker.go [hunk 2]
```diff
@@ -51,6 +51,8 @@ func (m Marker) String() string {
 	return m.comment + prefix + m.value
 }
 
+// CodeFragments represents a set of code fragments
```
Comment (camilamacedo86) [inline pkg/model/file/marker.go]: Could you please add more info? When and why the CodeFragments is used?

File: pkg/model/file/marker.go [hunk 3]
```diff
@@ -51,6 +51,8 @@ func (m Marker) String() string {
...
 type CodeFragments []string
 
+// CodeFragmentsMap binds CodeFragments to their Marker
```
Comment (camilamacedo86) [inline pkg/model/file/marker.go]: Could you please add more info? When and why the CodeFragments is used?
Would be great we doc why we have the  `CodeFragments []string` and `CodeFragmentsMap map[Marker]CodeFragments` as well.

File: pkg/model/file/interfaces.go (hunk 1 above)
Comment (Adirio) [reply pkg/model/file/interfaces.go]: I could find at least one case when it returns an error: `pkg/scaffold/internal/templates/v1/gopkg.go`. Skipping the rest of v1 files, will check v2 files now.

Comment (Adirio) [reply pkg/model/file/interfaces.go]: Found one file in v2 too: `pkg/scaffold/internal/templates/v2/kustomize.go`

**Comment by Adirio** [comment]
Hi @camilamacedo86

I rebased updating the documentation a bit. They are pretty short descriptions because they are part of the model package, they are not restricted to the current usage. That's the reason why I don't include information on how they are used in the model description. Further information on how they are used is provided in the `pkg/scaffold/model.Inserter` interface functions.

File: pkg/model/file/interfaces.go (hunk 1 above)
Comment (camilamacedo86) [inline pkg/model/file/interfaces.go]:
```suggestion
	// SetTemplateDefaults will return an error if not be possible to set the default values for templates
```
WDYT about something as ^?

Comment (Adirio) [reply pkg/model/file/interfaces.go]: IMHO that doesn't add any information as returning an `error` is part of the Go language.

Comment (camilamacedo86) [reply pkg/model/file/interfaces.go]: The comment is to describe what the func does. So, I understand that `it returns an error if not be possible to set the default values for templates`. Am I right? In this way, I did not get your comment.

**Comment by mengqiy** [comment]
It seems GH didn't get the signal that Travis has succeeded. :-/

**Comment by Adirio** [comment]
> It seems GH didn't get the signal that Travis has succeeded. :-/

I guess the easiest way to solve this is retriggering Travis

**Pull request merged**